Endpoint.swift

```diff
@@ -13,8 +13,18 @@ import Alamofire
 public enum EndpointSampleResponse {
     case Success(Int, NSData)
     case Error(Int?, NSError?, NSData?)
+    case Lazy(()-> EndpointSampleResponse)
+
+    func evaluate() -> EndpointSampleResponse {
+        switch self {
+        case Success, Error: return self
```

Comment:
Nice – didn't know you could use commas to delimit a list of case statements. I've been using `fallthrough` – this should clean up my code a bunch!